README.md

```diff
@@ -63,17 +63,19 @@ TEST RESULT
 ```
 Usage:
   treon
-  treon [PATH] [--threads=<number>] [-v]
+  treon [PATH] [--threads=<number>] [-v] [--exclude=<string>]...
```

Comment:
Let's just say,
`--exclude=<path>` instead of string here and below. To be more explicit.